components/Labels/ToggleNotify.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { c } from 'ttag';
+import { debounce } from 'proton-shared/lib/helpers/function';
+import { Toggle, useApiWithoutResult, useEventManager, useNotifications } from 'react-components';
+import { updateLabel } from 'proton-shared/lib/api/labels';
+
+const ToggleNotify = ({ label }) => {
+    const [toggled, setToggle] = useState(label.Notify === 1);
+    const { call } = useEventManager();
+    const { createNotification } = useNotifications();
+    const { request, loading } = useApiWithoutResult(updateLabel);
+
+    const handleChange = async () => {
+        const newLabel = {
+            ...label,
+            Notify: +!label.Notify
```

Comment:
Use the value coming from `event.target.checked`, it's safer if `label.Notify` change from somewhere.

Reply:
It's null and as it's a toggle !value works (if label changes, the toggle will too :) )

Reply:
The `Toggle` is a checkbox, so `checked` should be defined if you use `checked` property on the `Toggle`. It's always better to use the value coming from the event than the current state. Think the component as a controlled input.